src/output-json-ssh.h

```diff
@@ -28,6 +28,6 @@ void JsonSshLogRegister(void);
 
 #include "app-layer-ssh.h"
 
-void JsonSshLogJSON(json_t *js, SshState *tx);
+void JsonSshLogJSON(json_t *js, void *tx);
```

Comment:
Should I not remove this function ? How come CI passed when it seems still used by alert-prelude.c ?

Reply:
I don't think we enable prelude at compile time for any of the CI tests.  Probably should pick at least one build and turn it on.

Reply:
And compiler/linker do not complain if we declare functions but do not define them...